Installing Scrollmagic and gsap with npm/webpack

I have read and tried seemingly every approach that exists to import ScrollMagic with gsap while using webpack on the frontend, but I cannot get past this error for the life of me:
**(ScrollMagic.Scene) -> ERROR calling setTween() due to missing Plugin 'animation.gsap'. Please make sure to include plugins/animation.gsap.js**

Can you please share the current best practice for this, in terms of imports, aliases, and loaders?

@johanholm I've follow your advice but I still get the same error
```
ERROR in ./node_modules/scrollmagic/scrollmagic/uncompressed/plugins/animation.gsap.js
Module not found: Error: Can't resolve 'TimelineMax' in '../node_modules/scrollmagic/scrollmagic/uncompressed/plugins'
 @ ./node_modules/scrollmagic/scrollmagic/uncompressed/plugins/animation.gsap.js 31:2-61
 @ ./src/js/home.js
 @ ./src/index.js

ERROR in ./node_modules/scrollmagic/scrollmagic/uncompressed/plugins/animation.gsap.js
Module not found: Error: Can't resolve 'TweenMax' in '../node_modules/scrollmagic/scrollmagic/uncompressed/plugins'
 @ ./node_modules/scrollmagic/scrollmagic/uncompressed/plugins/animation.gsap.js 31:2-61
 @ ./src/js/home.js
 @ ./src/index.js
```

the error is **at line 31** of **animation.gsap.js**, this one:
`define(['ScrollMagic', 'TweenMax', 'TimelineMax'], factory);`

I don't know how to fix this on webpack., if anyone have a clue..

--- a/config/webpack.server.prod.js
+++ b/config/webpack.server.prod.js
@@ -82,19 +82,19 @@ module.exports = {
       path.resolve(__dirname),
       path.resolve(__dirname, '../node_modules')
     ],
-    alias: {
-      TweenLite: 'gsap/src/minified/TweenLite.min.js',
-      TweenMax: 'gsap/src/minified/TweenMax.min.js',
-      TimelineLite: 'gsap/src/minified/TimelineLite.min.js',
-      TimelineMax: 'gsap/src/minified/TimelineMax.min.js',
-      ScrollMagic: 'scrollmagic/scrollmagic/minified/ScrollMagic.min.js',
-      'animation.gsap': path.resolve(
-        'node_modules',
-        'scrollmagic/scrollmagic/minified/plugins/animation.gsap.min.js'
-      ),
-      'debug.addIndicators':
-        'scrollmagic/scrollmagic/minified/plugins/debug.addIndicators.min.js'
-    },
+    // alias: {
+    //   TweenLite: 'gsap/src/minified/TweenLite.min.js',
+    //   TweenMax: 'gsap/src/minified/TweenMax.min.js',
+    //   TimelineLite: 'gsap/src/minified/TimelineLite.min.js',
+    //   TimelineMax: 'gsap/src/minified/TimelineMax.min.js',
+    //   ScrollMagic: 'scrollmagic/scrollmagic/minified/ScrollMagic.min.js',
+    //   'animation.gsap': path.resolve(
+    //     'node_modules',
+    //     'scrollmagic/scrollmagic/minified/plugins/animation.gsap.min.js'
+    //   ),
+    //   'debug.addIndicators':
+    //     'scrollmagic/scrollmagic/minified/plugins/debug.addIndicators.min.js'
+    // },
     extensions: ['.js', '.jsx', '.json', '.scss']
   },
 

--- a/config/webpackConfigFactory.js
+++ b/config/webpackConfigFactory.js
@@ -159,34 +159,34 @@ module.exports = function(envType) {
 
   config.resolve = {
     alias: {
-      TweenLite: path.resolve(
-        '../node_modules',
-        'gsap/src/minified/TweenLite.min.js'
-      ),
-      TweenMax: path.resolve(
-        '../node_modules',
-        'gsap/src/minified/TweenMax.min.js'
-      ),
-      TimelineLite: path.resolve(
-        '../node_modules',
-        'gsap/src/minified/TimelineLite.min.js'
-      ),
-      TimelineMax: path.resolve(
-        '../node_modules',
-        'gsap/src/minified/TimelineMax.min.js'
-      ),
-      ScrollMagic: path.resolve(
-        '../node_modules',
-        'scrollmagic/scrollmagic/minified/ScrollMagic.min.js'
-      ),
-      'animation.gsap': path.resolve(
-        '../node_modules',
-        'scrollmagic/scrollmagic/minified/plugins/animation.gsap.min.js'
-      ),
-      'debug.addIndicators': path.resolve(
-        '../node_modules',
-        'scrollmagic/scrollmagic/minified/plugins/debug.addIndicators.min.js'
-      )
+      // TweenLite: path.resolve(
+      //   '../node_modules',
+      //   'gsap/src/minified/TweenLite.min.js'
+      // ),
+      // TweenMax: path.resolve(
+      //   '../node_modules',
+      //   'gsap/src/minified/TweenMax.min.js'
+      // ),
+      // TimelineLite: path.resolve(
+      //   '../node_modules',
+      //   'gsap/src/minified/TimelineLite.min.js'
+      // ),
+      // TimelineMax: path.resolve(
+      //   '../node_modules',
+      //   'gsap/src/minified/TimelineMax.min.js'
+      // ),
+      // ScrollMagic: path.resolve(
+      //   '../node_modules',
+      //   'scrollmagic/scrollmagic/minified/ScrollMagic.min.js'
+      // ),
+      // 'animation.gsap': path.resolve(
+      //   '../node_modules',
+      //   'scrollmagic/scrollmagic/minified/plugins/animation.gsap.min.js'
+      // ),
+      // 'debug.addIndicators': path.resolve(
+      //   '../node_modules',
+      //   'scrollmagic/scrollmagic/minified/plugins/debug.addIndicators.min.js'
+      // )
     },
     extensions: ['.js', '.jsx', '.json', '.scss']
   };

--- a/package.json
+++ b/package.json
@@ -93,6 +93,7 @@
     "response-time": "^2.3.2",
     "sass-loader": "^8.0.0",
     "scrollmagic": "^2.0.7",
+    "scrollscene": "0.0.12",
     "style-loader": "^1.0.0",
     "uglifyjs-webpack-plugin": "^2.2.0",
     "webpack": "^4.39.3",

--- a/src/pages/Home.js
+++ b/src/pages/Home.js
@@ -68,10 +68,10 @@
 import React, { Component } from 'react';
 //Import Components
 import Head from './Head';
-import Banner from '../components/homebanner/HomeBanner';
+import Banner from '../components/homeBanner/HomeBanner';
 import HomeCarousel from '../components/homecarousel';
 import PageAnimWrapper from '../components/pagetransition';
-import ScrollMagic from 'scrollmagic';
+import { ScrollScene } from 'scrollscene';
 
 class Home extends Component {
   render() {

